### PR TITLE
New Extension: Mouse Speed.

### PR DESCRIPTION
**Pull request by hammouda101010**
Mouse Speed is an extension that can get the speed of the mouse

### CODE REVIEW

**Comment by SharkPool-SP** [comment]
this could honeslty just be added as a new block to Sensing+

**Comment by CubesterYT** [comment]
Definitely not Sensing+ at this time, it's already bloated enough.

**Comment by hammouda101010** [comment]
> this could honeslty just be added as a new block to Sensing+, or any of the mouse extensions

that was what i was thinking!

**Comment by David-Orangemoon** [comment]
> Definitely not Sensing+ at this time, it's already bloated enough.

I think sensing + should just be hidden and deprecated.

**Comment by hammouda101010** [comment]
> > Definitely not Sensing+ at this time, it's already bloated enough.
> 
> I think sensing + should just be hidden and deprecated.

how about you rewrite it, its not THAT hard.

**Comment by David-Orangemoon** [comment]
> > > Definitely not Sensing+ at this time, it's already bloated enough.
> > 
> > I think sensing + should just be hidden and deprecated.
> 
> how about you rewrite it, its not THAT hard.

It was a bloated extension that should be replaced with other smaller extensions like multi touch, and a dedicated gyro extension.

**Comment by snowboyz0825** [comment]
I feel as though this is just bloat - you could code something that finds the mouse speed pretty easily, it would really only save ~5-10 blocks if coded well.

**Comment by hammouda101010** [comment]
> I feel as though this is just bloat - you could code something that finds the mouse speed pretty easily, it would really only save ~5-10 blocks if coded well.

well i tried to add more blocks so it doesnt looks like $h!11y bloat.

**Comment by hammouda101010** [comment]
> well i tried to add more blocks so it doesnt looks like $h!11y bloat.
>
i could also check if the mouse is faster than a sprite

**Comment by hammouda101010** [comment]
soo... will it get added or not?

**Comment by CubesterYT** [comment]
You must be patient, this repository is notoriously slow on review and merging, it's been months since a new extension has been merged.

**Comment by SharkPool-SP** [comment]
And it's gonna be 34 more

**Comment by hammouda101010** [comment]
> And it's gonna be 34 more

also only add mouse speed extension, im cooking a new extension rn

**Comment by hammouda101010** [comment]
> You must be patient, this repository is notoriously slow on review and merging, it's been months since a new extension has been merged.

turbowarp

File: extensions/Hammouda101010/turbomodz.js
Comment (hammouda101010) [inline extensions/Hammouda101010/turbomodz.js:1]: note: this extension is W.I.P. do not add it until its ready

**Comment by hammouda101010** [comment]
dang, its been a while

**Comment by David-Orangemoon** [comment]
> dang, its been a while

At least you hopefully won't have to wait 1/2 year for yours to get merged.

**Comment by hammouda101010** [comment]
please @GarboMuffin review this PR 😭

**Comment by CST1229** [comment]
please keep it to one extension per pull request. you can use [branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) in git to separate your extensions, and ideally your `master` branch should be left unmodified (as a base for your other branches)

**Comment by hammouda101010** [comment]
> please keep it to one extension per pull request. you can use [branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) in git to separate your extensions, and ideally your `master` branch should be left unmodified (as a base for your other branches)

well i modified the main branch lol but atleast it has only one extension this time

**Comment by hammouda101010** [comment]
calling reviewers @CubesterYT @CST1229

**Comment by Brackets-Coder** [comment]
> seriously, will this will never be reviewed for the sake of @GarboMuffin ?

Patience, young Padawan. Your extension will be reviewed in time. 

As of now, I'm pretty sure the repository merging format has been changed so that any PR that has been approved by two people with write access it will be automatically merged into the main branch. It looks like you don't have any reviews yet. 

Just be patient. This repository is notoriously slow...

**Comment by hammouda101010** [comment]
> > seriously, will this will never be reviewed for the sake of @GarboMuffin ?
> 
> Patience, young Padawan. Your extension will be reviewed in time.
> 
> As of now, I'm pretty sure the repository merging format has been changed so that any PR that has been approved by two people with write access will be automatically merged into the main branch. It looks like you don't have any reviews yet. As of now, I think this includes CST1229, CubesterYT, SharkPool-SP, DNin01, and of course GarboMuffin.
> 
> Just be patient. This repository is notoriously slow...

you're kidding me this will take a month for a reviewer to view this PR

**Review by unknown07724 — APPROVED** [review]
good thing

**Comment by hammouda101010** [comment]
they are conficts in the PR. can somebody fix them?

**Review by hammouda101010 — COMMENTED** [review]
everything look fine

**Comment by CubesterYT** [comment]
>
> you're kidding me this will take a month for a reviewer to view this PR

There are some PRs here that don't have reviews yet that are a year or two old, they haven't complained. Things take time, people have lives.

**Comment by hammouda101010** [comment]
it's been so long :skull: @GarboMuffin

**Comment by hammouda101010** [comment]
> Please do not add a random desktop.ini file and mp3. also it seems you used `npm install` which makes a new package-lock.
> 
> Is there any reason this cannot be merged with the existing cursor extension? https://extensions.turbowarp.org/cursor.js

i never made a desktop.ini file and never used `npm install`. plus the [cursor extension](https://extensions.turbowarp.org/cursor.js) is used to change the mouse cursor's appearence

**Comment by hammouda101010** [comment]
> I don't know why you closed this pull request but you did indeed somehow add a desktop.ini file. image and for cursor please look at #638
> 
> also do you plan on making a new pull request for this?

no

**Pull request closed without merging**